### PR TITLE
safely enforce sudo password prompting

### LINKED ISSUE #145: V-58901 sudo nopasswd too aggressive
The remediation removes the sudoers line entirely rather than just removing the NOPASSWD part, contrary to how the fix looks like it's written.

### PR DESCRIPTION
**Pull request by jamescassell**
fixes #145
fix taken from https://github.com/MindPointGroup/RHEL7-STIG/pull/73

### CODE REVIEW

**Comment by jamescassell** [comment]
Please test before merging.

File: tasks/cat2.yml [hunk 1]
```diff
@@ -2191,33 +2191,43 @@
...
-          state: absent
-      with_items: "{{ sudoer_configs }}"
+- name: "MEDIUM | V-58901 | The sudo command must require authentication."
+  block:
+      - name: "MEDIUM | V-58901 | AUDIT | The sudo command must require authentication."
+        # find included configs
+        shell: cat /etc/sudoers | grep -w '^#include' | awk '{print $2}'
+        check_mode: no
+        changed_when: no
+        register: sudoers_include_audit
```
Comment (defionscode) [inline tasks/cat2.yml]: i dont believe we wanted to continue to have audit-only tasks..it doesn't _seem_ like this is used anywhere, am I mistaken?

Comment (jamescassell) [reply tasks/cat2.yml]: Looks like it is looking for included files and using it below. We aren't this aggressive in the role for 7.

**Comment by jamescassell** [comment]
I've done some moderate testing on this PR and it seems to work for the cases I've tried, so unless there are concerns, I'd merge it.  It's doubly disabled by default, in any case.

The old version removed any sudoers entry entirely, locking out a user who did have a password available to use w/ sudo.

**Comment by jamescassell** [comment]
I'll merge this in a week unless there are any concerns.

**Pull request merged**